<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>exportDialog</class>
 <widget class="QDialog" name="exportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>639</width>
    <height>273</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export parameters</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>images/uicilibris.svg</normaloff>images/uicilibris.svg</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Mediawiki's URL</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="urlEdit">
        <property name="toolTip">
         <string>URL of the mediawiki's homepage. Begins with 'http://'</string>
        </property>
        <property name="statusTip">
         <string>URL of the mediawiki's homepage. Begins with 'http://'</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string notr="true">http://</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Name Space</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="nsEdit">
        <property name="toolTip">
         <string>A short word with a colon</string>
        </property>
        <property name="statusTip">
         <string>A short word with a colon</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>New:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Validation</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Login</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Password</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="loginEdit">
        <property name="toolTip">
         <string>Your login in this mediawiki</string>
        </property>
        <property name="statusTip">
         <string>Your login in this mediawiki</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="passwdEdit">
        <property name="toolTip">
         <string>Your password</string>
        </property>
        <property name="statusTip">
         <string>Your password</string>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QWidget" name="widget_2" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QToolButton" name="validButton">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../../../../../../usr/share/icons/Tango/scalable/categories/applications-system.svg</normaloff>../../../../../../usr/share/icons/Tango/scalable/categories/applications-system.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>64</width>
             <height>64</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="loggedButton">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../../../../../../usr/share/icons/Tango/scalable/emotes/face-monkey.svg</normaloff>../../../../../../usr/share/icons/Tango/scalable/emotes/face-monkey.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>64</width>
             <height>64</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="checkButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>Make a few checks in the target mediawiki</string>
           </property>
           <property name="statusTip">
            <string>Make a few checks in the target mediawiki</string>
           </property>
           <property name="text">
            <string>Make checks</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>exportDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>exportDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
